docs/satellite-configuration-tuning.rst

```diff
@@ -67,9 +67,31 @@ Once the file has been edited, the following commands need to be run to make the
 qdrouterd and qpid tunings
 ==========================
 
+Disabling Katello Agent Infrastructure on Satellite 6.10+
+---------------------------------------------------------
+
+Katello-agent infrastructure is used to push package updates to Content Hosts from Satellite. This uses Apache Qpid and Qpid Dispatch Router (qdrouterd) for messaging between Satellite, Capsules, and Content Hosts. Starting with Satellite 6.10, Katello Agent Infrastructure is optional and disabled by default for new installations. For Satellite deployments upgraded from prior versions, it will remain enabled upon upgrade; however if this feature is not being used, it is recommended to disable it to conserve hardware resources on Satellite and Capsule servers.
+
+To disable and remove katello-agent infrastructure on (upgraded) Satellite 6.10+, run the command:
+
+  `satellite-installer --katello-enable-katello-agent false`
```

Comment:
This should be:

```
  satellite-installer --foreman-proxy-content-enable-katello-agent false
```

We made it so the command was the same whether on Satellite or a Capsule. You can see this for example [here](https://github.com/theforeman/forklift/blob/master/bats/fb-katello-client.bats#L15).

Reply:
Thanks, I addressed this by consolidating the instructions for Satellite and Capsule.